components/Buttons/src/Theming/MDCFloatingButton+Theming.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialButtons.h"
+#import "MaterialContainerScheme.h"
+
+#import <Foundation/Foundation.h>
+
+@interface MDCFloatingButton (MaterialTheming)
+
+/**
+ Apply the secondary theme to this MDCFloatingButton instances.
```

Comment:
Nit: Please clean-up the comment a little bit.  Suggestion: "Applies the secondary Floating Action Button style to an MDCFloatingButton instance".

Reply:
Addressed. Thanks for the suggestion.